### DISCUSSION TITLE
When drawing in parallel, is the cache file larger than 400G?

### DISCUSSION THREAD

**Discussion by Aunt-Lee** [Chia-Network/chia-blockchain#1630] [Q&A]
Will the cached files grow as more files are drawn in parallel?

**Comment by hoffmang9** [comment]
Each plotting process takes its own 356 GB of temp space at its peak need of space.

**Comment by Aunt-Lee** [comment]
If I have two plots in parallel, will it be necessary to prepare storage in excess of 700g, right?




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Chia-Network/chia-blockchain"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年4月1日(星期四) 中午1:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [Chia-Network/chia-blockchain] When drawing in parallel, is the cache file larger than 400G? (#1630)





 
Each plotting process takes its own 356 GB of temp space at its peak need of space.

**Reply by pieterhelsen** [reply]
Yes. That's correct.
